src/main/java/calculator/deque/ElementDeque.java

```diff
@@ -0,0 +1,57 @@
+package calculator.deque;
+
+import calculator.element.Element;
+import calculator.element.operand.Operand;
+import calculator.element.operaor.Operator;
+import calculator.element.operaor.OperatorFactory;
+
+import java.util.*;
+
+public class ElementDeque {
+    Deque<Element> elements = new ArrayDeque<>();
+
+    public ElementDeque(String[] splitInput) {
+        for (int index = 0; index < splitInput.length; index++) {
+            addElement(splitInput[index], index);
+        }
+    }
+
+    private void addElement(String elementStr, int index) {
+        if(isOperandPosition(index)) {
+            addOperand(elementStr);
+        } else {
+            addOperator(elementStr);
+        }
+    }
+
+    private void addOperand(String elementStr) {
```

Comment:
```suggestion
    private void addOperand(String element) {
```
이미 타입으로 String을 표시하고 있으니 변수명에 반복할 필요는 없어보입니다

Reply:
이거 버릇인데 고쳐야겠네요. 감사합니다.